src/components/Tabs/index.tsx

```diff
@@ -5,14 +5,10 @@ import {
 	Tabs as GrommetTabs,
 	TabsProps,
 } from 'grommet';
-export { TabProps, TabsProps } from 'grommet';
 import React from 'react';
 import asRendition from '../../asRendition';
 
 interface InnerTabProps extends TabProps {
-	compact?: boolean;
```

Comment:
Wait, why is this being removed?

Reply:
@StefKors after chatting with Andrea it made sense, he added the compact CSS as part of the `Tabs` component, so we can stop doing `children.map`.

Reply:
The `compact` property is usable only on the tabs component and that property is used to make some graphic changes. By adding the Tab style as a nested Tabs style gives the same result and allow to use the property only in one component.